modules/@apostrophecms/characterCount/index.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
         const finalTools = {
           ...initialData.tools,
           characterCount: {
-            component: 'TiptapCharacterCount',
+            component: 'AposCCToolbar',
```

Comment:
If this isn't going to ship as a feature per se, then we shouldn't use the Apos prefix as that could cause the example to collide with something we release later.

Reply:
Changed.